src/test/index.ts

```diff
@@ -364,7 +455,11 @@ export const test = _baseTest.extend<TestFixtures, WorkerAndFileFixtures>({
     }
   },
 
-  page: async ({ context }, use) => {
+  page: async ({ context, _reuseBrowerContext }, use) => {
+    if (_reuseBrowerContext.isEnabled() && _reuseBrowerContext.hasContext()) {
```

Comment:
If _reuseBrowerContext.isEnabled() is true and given that we depend on context fixture, how can this be that hasContext is false?